.github/ISSUE_TEMPLATE/not-meeting.yml

```diff
@@ -0,0 +1,18 @@
+name: Group not meeting
+description: Let the W3C staff know your group is not planning to meet.
+labels: ["not-meeting"]

```

Comment:
The label seems good. On top of that, I'm thinking that, for our own sanity, for example to avoid ending up with a list of conflicts that contains issues of groups that are **not** meeting, we should also close the issue so that it does not show in listings by default.

Perhaps worth announcing that in the template? Something like "We're going to close this issue, don't worry, that's on purpose". With better wording ;)

Reply:
Added some wording